src/cli/peakdetector/peakdetector.pro

```diff
@@ -55,12 +57,14 @@ macx {
 SOURCES	= options.cpp                                            \
           $$top_srcdir/src/core/libmaven/classifier.cpp          \
           $$top_srcdir/src/core/libmaven/classifierNeuralNet.cpp \
+          $$top_srcdir/src/gui/mzroll/downloadmanager.cpp        \
           main.cpp                                               \
           parseoptions.cpp                                       \
           peakdetectorcli.cpp
 
 HEADERS += $$top_srcdir/src/core/libmaven/classifier.h          \
            $$top_srcdir/src/core/libmaven/classifierNeuralNet.h \
+           $$top_srcdir/src/gui/mzroll/downloadmanager.h        \
```

Comment:
I would suggest that we put `downloadmanager` in PollyCLI module instead. It fits as a common set of utilities to be shared by other modules.

Reply:
Since `downloadmanager` might be used for version upgrade etc as well, it should be a separate module instead of pollyCLI. If @rish9511 only plans to use this for polly, then let's move it to pollyCLI

Reply:
Makes sense. It should still not be part of the GUI module. Maybe move it to libmaven? But then we will have to find not-Qt alternatives to QNetwork type of functionality.

Reply:
It can be an independent module like pollyCLI and projectDB. Does not need to go into libmaven, necessarily.

Reply:
In that case, this new module can merge with pollyCLI. Then the module will accommodate classes responsible for making requests over the network.

Reply:
Let's make a new folder called "common" ?
Even PollyCLI  can be part of it and can we save emdb  format from cli ?

Reply:
Sounds good! If it can be done by EoD, let's do it. If not, we can add it to the next release

Reply:
Done